### DISCUSSION TITLE
QA 기간을 더 줄이고 싶어요.

### DISCUSSION THREAD

**Discussion by zangsu** [woowacourse-teams/2024-code-zap#391] [이야기해봐요]
## 이유

1. 이번 스프린트부터 개발 서버가 주기적으로 배포가 되기 때문에 각 기능에 대해 굳이 QA 기간이 아니라도 팀 단위로 테스트를 진행할 수 있는 기간이 충분합니다.
2. 실제 사용할 수 있는 공식 개발 기간이 9일 (데모데이는 제외) 인데, 이 중 이틀을 QA 기간에 사용하면 기능 개발에 부담이 커진다.

## 제안 사항

QA 기간을 목요일 1시 (or 3시) 시작 정도로 두고, 다 같이 예상되는 케이스를 확인해 봅시다. 
+) 개발 서버에 코드가 머지되고 나면 팀 단위로 각자 테스트를 진행하면 좋을 것 같아요.

> 솔라는 개발 종료 날짜를 "celebration day" 로 명명하고, 팀 단위로 축하하는 날을 가졌다고 해요.

**Comment by Hain-tain** [comment]
< 이전에 결정했던 코드잽 2주 스프린트 간의 일정>

1주차 기간

월, 화, 수 - 각자 기능 개발

목 16시 - 중간 점검 (Z 스프린트)

금 - 리뷰 + 각자 기능 개발

2주차 기간

월요일 밤 23시 59분 - PR 올리기

화요일 18시 - 리뷰 + PR 머지 + 배포 (A 스프린트)

수, 목 23시 - QA 문제 확인 및 문제해결

금 - 데모데이+ 회식 😎 - (P 스프린트)

**Comment by zangsu** [comment]
우선은 QA 에 대한 필요성을 크게 느끼고 있는 팀원이 많아 현상 유지 하겠습니다.
